apps/api/src/domain/models/playlist/commands/create-playlist.command.ts

```diff
@@ -3,6 +3,21 @@ import { Command } from '@coscrad/commands';
 import { ExternalEnum, NestedDataType, NonEmptyString, UUID } from '@coscrad/data-types';
 import { AggregateCompositeIdentifier } from '../../../types/AggregateCompositeIdentifier';
 
+export const LanguageCodeEnum = ExternalEnum(
```

Comment:
We should export this from one place (maybe `multilingual-text.entity.ts`). I have a pattern.
```ts
export const LanguageCode = (options) =>  ExternalEnum(
    {
        enumLabel: `language code`,
        enumName: `LanguageCode`,
        labelsAndValues: Object.entries(LanguageCode).map(([label, languageCode]) => ({
            label,
            value: languageCode,
        })),
    },
   options
)
```

Reply:
done